sdk/CoreWCF/Azure.CoreWCF.AzureQueueStorage/Azure.CoreWCF.AzureQueueStorage.sln

```diff
@@ -0,0 +1,81 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Unless we're planning on strictly adopting the API patterns in the [Azure SDK Guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html), these packages cannot use the `Azure.*` prefix.   We'll need to move them over to `Microsoft.Azure.*`.

If we are planning to adhere to the SDK guidelines, we'll need to start a conversation with the archboard - particularly @KrzysztofCwalina and @tg-msft - around the namespace.   `CoreWCF` is not currently an approved service area.